### ISSUE TITLE
Short Install Guide - for same machines

### ISSUE THREAD

**Issue by DAlexis74** [seven-of-eleven/Lenovo-ThinkPad-L13-Yoga-Hackintosh#5]
It was a great pleasure to see that the usb installer is working on the same macine just havin i7 processor - built with hard work based on the forums and open core documentation. On the other hand, It would be great to see thet this setup is shared better as I have issue to start the second phase of the installer, after the first reboot. The preinstallaton environment does not start.

**Comment by seven-of-eleven** [comment]
You may want to change the config.plist file to show the boot picker menu when you boot up. Then for each restart you'll need to use the boot menu with F12, select the USB key and then you should be able to select the proper install from the opencore menu.  I didn't really want to provide an installation guide as that can be found on the OpenCore site but I'll add this when I update to OC 0.6.9 when it comes out next month (I usually skip one or two versions).

**Comment by DAlexis74** [comment]
Thank you very much, I found these options, on the other hand I had to read a lot of pages without any reason. 
That would be great to see, or just turning on in the files the boot menu.
Here I would not write a huge documentatation, just a short script. 
0. The prerequisites are well described.
1. How to prepare the install media (link the relevant page - do not desribe it)
2. How to mount EFI partition and put the boot files on it - mentioning the files from the repository.

On the other hand - for me after the first reboot, the system did not complete the installation (crashed adter realtek 8153 pathing) - I used the repository linked: hagenest/thinkpad-l13-yoga-hackintosh
